pysteps/nowcasts/anvil.py

```diff
@@ -0,0 +1,370 @@
+"""
+pysteps.nowcasts.anvil
+======================
+
+Implementation of the autoregressive nowcasting using VIL (ANVIL) nowcasting
+method developed in :cite:`PCHL2020`. ANVIL is an extension of S-PROG. The
+main improvements are using an autoregressive integrated (ARI) model and
+vertically integrated liquid (VIL) as the input variable. In addition, the ARI
+model allows localization of the parameter estimates.
+
+.. autosummary::
+    :toctree: ../generated/
+
+    forecast
+"""
+
+import sys
+import time
+import numpy as np
+from scipy.ndimage import gaussian_filter
+from pysteps import cascade, extrapolation
+from pysteps.timeseries import autoregression
+from pysteps import utils
+
+try:
+    import dask
+
+    DASK_IMPORTED = True
+except ImportError:
+    DASK_IMPORTED = False
+
+
+def forecast(vil, rainrate, velocity, n_timesteps, n_cascade_levels=8,
+             extrap_method="semilagrangian", ar_order=2, ar_window_radius=50,
+             r_vil_window_radius=5, fft_method="numpy", num_workers=1,
+             extrap_kwargs=None, filter_kwargs=None, measure_time=False):
+    """Generate a nowcast by using the ANVIL method.
```

Comment:
Perhaps a very short description of the method could be added to the docstrings.